docs/jobs/openings/EA.md

```diff
@@ -0,0 +1,37 @@
+## Executive Assistant opening @ Aragon (Zug, Switzerland)
+
+Aragon is an open source, decentralized project [fighting for the freedom](https://www.youtube.com/watch?v=AqjIWmiAidw) to organize. We believe that decentralized organizations can solve the world's worst problems.
```

Comment:
> Aragon is an open source, decentralized project [fighting for the freedom](https://www.youtube.com/watch?v=AqjIWmiAidw) to organize.

I guess there should be something after _to organize_?

Reply:
All the openings have the same line, I think the source is our website actually. What would you add? Maybe `fighting for the freedom of organization`?

Reply:
My bad, LGTM